libs/pages/services/src/lib/feature/deployments/deployments-feature.tsx

```diff
@@ -1,22 +1,20 @@
 import {
   environmentFactoryMock,
   environmentsLoadingEnvironmentDeployments,
+  environmentsLoadingStatus,
   fetchEnvironmentDeploymentHistory,
-  fetchEnvironments,
   selectEnvironmentById,
-  environmentsLoadingStatus,
 } from '@console/domains/environment'
 import { DeploymentService, EnvironmentEntity } from '@console/shared/interfaces'
 import { BaseLink } from '@console/shared/ui'
 import { AppDispatch, RootState } from '@console/store/data'
-import { DeploymentsPage } from '@console/pages/services/ui'
 import { useEffect } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 import { useParams } from 'react-router'
 import { ServicesEnum } from '@console/shared/enums'
-import { DeploymentHistoryApplication } from 'qovery-typescript-axios'
+import DeploymentsPage from '../../ui/deployments-page/deployments-page'
 
-export function Deployments() {
+export function DeploymentsFeature() {
```

Comment:
Here, Deployments becomes DeploymentsFeature

Reply:
It's necessary to have `deployments-feature.tsx` ? we can have directly `deployments.tsx`?